package/MDAnalysis/topology/RDKitParser.py

```diff
@@ -0,0 +1,282 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
new modules will need a .rst stub

Reply:
I'm not too familiar with sphinx yet, are these 2 files the only thing I need to create/modify ?
package/doc/sphinx/source/documentation_pages/topology/RDKitParser.rst
package/doc/sphinx/source/documentation_pages/topology_modules.rst